data_iterator.go

```diff
@@ -25,7 +24,7 @@ type DataIterator struct {
 	logger         *logrus.Entry
 }
 
-func (d *DataIterator) Run() {
+func (d *DataIterator) Run(tables []*schema.Table) {
```

Comment:
Why this change?

Reply:
I wanted to refactor the table loading logic from `Start` to `Initialize`. I thought I took this part out, but I guess I didn't do it fully. 

As stated [earlier](https://github.com/Shopify/ghostferry/pull/89#discussion_r259859369), I'll refactor this part anyway, so I'm not going to remove it from this PR anymore.